### PR TITLE
Fix #98 - remove c_code after build

### LINKED ISSUE #98: Clean up after building a version to save space
Talked to someone with not a whole lot of space left on their device that wanted to give Nim a try today. Checked how much `choosenim` was using on my system and it showed a pretty steep 4.1G of used space. I've got about 9 versions installed, but 0.19.0 alone is almost half a gig large. The main culprit for each version appears to be the c_code folder. The downloads folder is another half gig combined. It would be nice if choosenim could clean up these files that are no longer used for anything so that it didn't hog as much space.

### PR DESCRIPTION
**Pull request by genotrance**

### CODE REVIEW

File: src/choosenim/builder.nim [hunk 1]
```diff
@@ -110,6 +110,11 @@ proc build*(extractDir: string, version: Version, params: CliParams) =
     buildTools()
     when defined(posix):
       setPermissions() # workaround for #147
+    # Delete c_code
+    try:
+      removeDir(extractDir / "c_code")
+    except OSError:
+      discard
```
Comment (dom96) [inline src/choosenim/builder.nim]: Move this to the `finally` instead below.

File: src/choosenim/builder.nim [hunk 2]
```diff
@@ -85,6 +85,7 @@ when defined(posix):
                             fpGroupRead, fpGroupExec,
                             fpOthersRead, fpOthersExec}
         )
+        display("Info", "Setting execution permissions", Message, LowPriority)
```
Comment (dom96) [inline src/choosenim/builder.nim]: please include the path in this message

File: src/choosenim/builder.nim [hunk 3]
```diff
@@ -120,6 +121,12 @@ proc build*(extractDir: string, version: Version, params: CliParams) =
     raise newError
   finally:
     if success:
+      # Delete c_code
+      try:
+        removeDir(extractDir / "c_code")
+      except OSError:
+        discard
```
Comment (dom96) [inline src/choosenim/builder.nim]: Also, add a warning here as done in the `try` that's also removing stuff below please.

File: src/choosenim/builder.nim (hunk 1 above)
Comment (genotrance) [reply src/choosenim/builder.nim]: Done

File: src/choosenim/builder.nim (hunk 2 above)
Comment (genotrance) [reply src/choosenim/builder.nim]: Done

File: src/choosenim/builder.nim (hunk 3 above)
Comment (genotrance) [reply src/choosenim/builder.nim]: Done

**Pull request merged**